init.lua

```diff
@@ -146,7 +151,11 @@ local lootData = {}
 local doSell = false
 local cantLootList = {}
 local cantLootID = 0
-
+local tmpDoLoot = false
+local questItemCount = {
```

Comment:
how does this get set to something? are you just manually updating this in the code to some item name and number if you're collecting something for a quest?

Reply:
that was something I started to work on to get quest counting to work. 
I meant to take that out of the commit. 
right now it does nothing.